templates/docs/patterns/chip.md

```diff
@@ -0,0 +1,30 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Chip | Components
+---
+
+## Chips
+
+<hr>
+
+Our chip component is to be used to display small actionable pieces of information.
+
+<div class="embedded-example"><a href="/docs/examples/patterns/chip/" class="js-example">
+View example of the default chip pattern
+</a></div>
+
+### Import
+
+To import just this component into your project, copy the snippet below and include it in your main Sass file.
+
+```scss
+@import 'patterns_chip';
+@include vf-p-chip;
+```
+
+For more information see [Customising Vanilla](/docs/customising-vanilla/) in your projects, which includes overrides and importing instructions.
+
+### Design
+
+Unfortunately there are no design specs available for this component.
```

Comment:
Noted, I'll add design spec ASAP. Issue here: https://github.com/canonical-web-and-design/vanilla-framework/issues/3161

Reply:
Thanks, I didn't know if you wanted to do them anymore.

Reply:
@anthonydillon 

We talked with @lyubomir-popov and @spencerbygraves if there is any need to maintain the design (in a repo) at all.
The design team is creating the specs in Zeplin, so they are available for devs during the component build, and the design spec repo seems to be unmaintained and not really used by anyone.

So we thought that probably it's not worth maintaining it, and adding new designs there.

In this case we would just remove the `Design` section from this docs page completely, and later we could archive the design spec repo and maybe remove the Design section from other docs pages as well.

Any thoughts?

Reply:
Ok if that is the way in which the design team want to continue to drive the framework that's fine by me. I think we can archive the repo now and start removing the design sections soon.